packages/magic-entries-webpack-plugin/package.json

```diff
@@ -29,7 +29,7 @@
     "tslib": "^1.14.1"
   },
   "peerDependencies": {
-    "webpack": ">4.25.1, <=4.5.0"
+    "webpack": ">4.25.1 <=4.5.0"
```

Comment:
I think this needs to be `^4.25.1` The current webpack version on sewing-kit is `4.44.1`.
According to the tool you pointed out. https://semver.npmjs.com/ `^4.25.1`  will match the versions that are supported.

Reply:
Hm, I think you're right. It doesn't like the range.